fail2ban/server/database.py

```diff
@@ -34,6 +34,14 @@
 from .ticket import FailTicket
 from ..helpers import getLogger
 
+try:
+	import sqlitebck
```

Comment:
Ok understood. I am a Fedorian myself and at the moment it is not packaged there either, but I will do that if this patch gets accepted.
As it is written it adds a new feature, but only if sqlitebck is available. Another possibility would be to copy the database another way if sqlitebck is not available (https://docs.python.org/release/2.6/library/sqlite3.html#sqlite3.Connection.iterdump)
As to the benefit: When we started using fail2ban we saw the number of writes/sec to disk increase from an average of 100 to an average of 600. This dropped again to 100 when we switched to :memory: (but this causes minutes of 100% CPU on startup)
Anyway, thanks for responding!